docs/resources/aviatrix_firenet.md

```diff
@@ -61,8 +61,13 @@ The following arguments are supported:
 * `east_west_inspection_excluded_cidrs` - (Optional) Network List Excluded From East-West Inspection. CIDRs to be excluded from inspection. Type: Set(String). Available as of provider version R2.19.5+.
 * `tgw_segmentation_for_egress_enabled` - (Optional) Enable TGW segmentation for egress. Valid values: true or false. Default value: false. Available as of provider version R2.19+.
 * `hashing_algorithm` - (Optional) Hashing algorithm to load balance traffic across the firewall. Valid values: "2-Tuple", "5-Tuple". Default value: "5-Tuple".
+
 * `keep_alive_via_lan_interface_enabled` - (Optional) Enable Keep Alive via Firewall LAN Interface. Valid values: true or false. Default value: false. Available as of provider version R2.18.1+.
 
+-> **NOTE:** `keep_alive_via_lan_interface_enabled` - Option to enable/disable keep_alive_via_lan_interface will be removed in Aviatrix provider v3.2.0. As of provider v3.1.2, disabling keep_alive_via_lan_interface will NOT be allowed.
+If you have set 'keep_alive_via_lan_interface_enabled = true', no action is needed at this time. After you upgrade to Aviatrix provider v3.2.0, you can safely remove the 'keep_alive_via_lan_interface_enabled' attribute from your configuration.
+If you have set 'keep_alive_via_lan_interface_enabled = false', you must enable it before you can upgrade to Aviatrix provider v3.2.0.
```

Comment:
```suggestion
```
I think its better to have these instructions in the changelist I will add later closer to release date. This is too much information inside the box and will not fit properly and be hard to read